packages/interface/src/components/complementary-area/index.js

```diff
@@ -204,59 +249,56 @@ function ComplementaryArea( {
 					{ title }
 				</ComplementaryAreaMoreMenuItem>
 			) }
-			{ isActive && (
```

Comment:
This condition moved within the "fill" to ensure the AnimatePresence animation work. This is probably the most impactful change here that we need to assess in terms of performance...